app/src/main/java/com/ciandt/dragonfly/example/features/login/LoginActivity.kt

```diff
@@ -102,12 +103,15 @@ class LoginActivity : BaseActivity(), LoginContract.View, GoogleApiClient.OnConn
         progress.makeVisible()
     }
 
-    override fun showError() {
+    override fun showError(exception: Exception) {
+        DragonflyLogger.warn(LOG_TAG, exception.message ?: "")
```

Comment:
I removed DragonflyLogger classes from Presenter because it contains Android code. We need to find a better option to inject our Logger class.